package.json

```diff
@@ -294,14 +296,15 @@
     "react-dom": "^16.3.0",
     "react-redux": "^5.0.6",
     "redux": "^4.0.0",
+    "sinon": "^7.2.4",
```

Comment:
interesting that TS doesn't complain about missing `@types/sinon` anymore

Reply:
🤷‍♂️